CHANGES.rst

```diff
@@ -12,6 +12,22 @@ Yet to be released.
 
 - [Bugfix] #15 #17 - Restore Python 2.6 compatibility. (Thanks to Marco Nenciarini!)
 
+.. attention::
+    Deprecation warnings, these symbols are slated for removal in
+    the next major release.
```

Comment:
This sentence reads weird IMHO (it starting with "Deprecation warnings, ..."). I also don't think "symbols" is a common term for Python.

What about: "Deprecation warning: These functions and attributes are slated for ..."?